test/common/scale_up_replicas_threescale.go

```diff
@@ -76,48 +75,24 @@ func getAPIManager(dynClient k8sclient.Client) (threescalev1.APIManager, error)
 }
 
 func updateAPIManager(dynClient k8sclient.Client, replicas int64) (threescalev1.APIManager, error) {
-
 	apim, err := getAPIManager(dynClient)
 	if err != nil {
 		return apim, err
 	}
-	resourceRequirements := true
-	apim = threescalev1.APIManager{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:            name,
-			Namespace:       namespace,
-			ResourceVersion: apim.GetResourceVersion(),
-		},
-		Spec: threescalev1.APIManagerSpec{
-			HighAvailability: &threescalev1.HighAvailabilitySpec{},
-			APIManagerCommonSpec: threescalev1.APIManagerCommonSpec{
-				ResourceRequirementsEnabled: &resourceRequirements,
-			},
-			System: &threescalev1.SystemSpec{
-				DatabaseSpec: &threescalev1.SystemDatabaseSpec{
-					PostgreSQL: &threescalev1.SystemPostgreSQLSpec{},
-				},
-				FileStorageSpec: &threescalev1.SystemFileStorageSpec{
-					S3: &threescalev1.SystemS3Spec{},
-				},
-				AppSpec:     &threescalev1.SystemAppSpec{Replicas: &[]int64{replicas}[0]},
-				SidekiqSpec: &threescalev1.SystemSidekiqSpec{Replicas: &[]int64{replicas}[0]},
-			},
-			Apicast: &threescalev1.ApicastSpec{
-				ProductionSpec: &threescalev1.ApicastProductionSpec{Replicas: &[]int64{replicas}[0]},
-				StagingSpec:    &threescalev1.ApicastStagingSpec{Replicas: &[]int64{replicas}[0]},
-			},
-			Backend: &threescalev1.BackendSpec{
-				ListenerSpec: &threescalev1.BackendListenerSpec{Replicas: &[]int64{replicas}[0]},
-				WorkerSpec:   &threescalev1.BackendWorkerSpec{Replicas: &[]int64{replicas}[0]},
-				CronSpec:     &threescalev1.BackendCronSpec{Replicas: &[]int64{replicas}[0]},
-			},
-			Zync: &threescalev1.ZyncSpec{
-				AppSpec: &threescalev1.ZyncAppSpec{Replicas: &[]int64{replicas}[0]},
-				QueSpec: &threescalev1.ZyncQueSpec{Replicas: &[]int64{replicas}[0]},
-			},
-		},
-	}
+
+	apim.Name = name
+	apim.Namespace = namespace
+	apim.ResourceVersion = apim.GetResourceVersion()
```

Comment:
Is the name, namespace and resourceversion added intentionally here?
I would expect those field to already be set in the CR we Get. 🤔

Reply:
This is minor, and can be left as is.
I am approving, but it will merge after the release.

Reply:
I would also expect them to be present already. I can try to remove and rerun the tests.